test/e2e/framework/network/utils.go

```diff
@@ -380,17 +380,20 @@ func (config *NetworkingTestConfig) GetHTTPCodeFromTestContainer(path, targetIP
 	return code, nil
 }
 
-// DialFromNode executes a tcp or udp request based on protocol via kubectl exec
+// DialFromNode executes a tcp or udp curl/echo request based on protocol via kubectl exec
 // in a test container running with host networking.
-// - minTries is the minimum number of curl attempts required before declaring
-//   success. Set to 0 if you'd like to return as soon as all endpoints respond
-//   at least once.
-// - maxTries is the maximum number of curl attempts. If this many attempts pass
-//   and we don't see all expected endpoints, the test fails.
-// maxTries == minTries will confirm that we see the expected endpoints and no
-// more for maxTries. Use this if you want to eg: fail a readiness check on a
-// pod and confirm it doesn't show up as an endpoint.
-func (config *NetworkingTestConfig) DialFromNode(protocol, targetIP string, targetPort, maxTries, minTries int, expectedEps sets.String) {
+// - minTries is the minimum number of curl/echo attempts required before declaring
+//   success. If 0, then we return as soon as all endpoints succeed.
+// - There is no logical change to test results if faillures happen AFTER endpoints have succeeded,
+//   hence over-padding minTries will NOT reverse a successful result and is thus not very useful yet
+//   (See the TODO about checking probability, which isnt implemented yet).
+// - maxTries is the maximum number of curl/echo attempts before an error is returned.  The
+//   smaller this number is, the less 'slack' there is for declaring success.
+// - if maxTries < expectedEps, this test is guaranteed to return an error, because all endpoints wont be hit.
+// - maxTries == minTries will return as soon as all endpoints succeed (or fail once maxTries is reached without
+//   success on all endpoints).
+//   In general its prudent to have a high enough level of minTries to guarantee that all pods get a fair chance at receiving traffic.
```

Comment:
Given all the comment space that is spent on minTries/maxTries, would it be prudent to log those in test output?  Right now only maxTries is logged, and only on error

Reply:
fixeddddd, added a log message below